python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -221,7 +222,11 @@ def to_datetime(
                 format=format,
             )
             return as_index(col, name=arg.name)
-        elif isinstance(arg, cudf.Series):
+        elif isinstance(arg, (cudf.Series, pd.Series)):
+            if isinstance(arg, (pd.Series)):
+                # this needs to work for a pandas series
+                # input or else it becomes an index
+                arg = cudf.from_pandas(arg)
             col = arg._column
```

Comment:
Is the `as_column` machinery a no-op if it's already a cudf column (series)? Perhaps we could rely on that here if so?

Reply:
I'll try this out, it should be able to work.